kernel/src/common/hw_handlers.c

```diff
@@ -121,44 +122,53 @@ long __attribute__((interrupt("SWI"))) software_interrupt_handler(void)
 		break;
 	case SYSCALL_CREATE:
 		os_printf("Create system call called!\n");
+		return -1;
 
-		return (long) kcreate((char*) r0, r1, 0);
+//		return (long) kcreate((char*) r0, r1, 0);
 	case SYSCALL_DELETE:
 		os_printf("Delete system call called!\n");
+		return -1;
 
-		return (long) kdelete((char*) r0, 1);
+//		return (long) kdelete((char*) r0, 1);
 	case SYSCALL_OPEN:
 		os_printf("Open system call called!\n");
+		return -1;
 
-		return (long) kopen((char*) r0, r1);
+//		return (long) kopen((char*) r0, r1);
 	case SYSCALL_MKDIR:
 		os_printf("Mkdir system call called!\n");
+		return -1;
 
-		return (long) kcreate((char*) r0, 'w', 1);
+//		return (long) kcreate((char*) r0, 'w', 1);
 	case SYSCALL_READ:
 		os_printf("Read system call called!\n");
+		return -1;
 
-		return (long) kread(r0, (void*) r1, r2);
+//		return (long) kread(r0, (void*) r1, r2);
 	case SYSCALL_WRITE:
 		os_printf("Write system call called!\n");
+		return -1;
 
-		return (long) kwrite(r0, (void*) r1, r2);
+//		return (long) kwrite(r0, (void*) r1, r2);
 	case SYSCALL_CLOSE:
 		os_printf("Close system call called!\n");
+		return -1;
 
-		return (long) kclose(r0);
+//		return (long) kclose(r0);
 	case SYSCALL_SEEK:
 		os_printf("Seek system call called!\n");
+		return -1;
 
-		return (long) kseek(r0, r1);
+//		return (long) kseek(r0, r1);
 	case SYSCALL_COPY:
 		os_printf("Copy system call called!\n");
+		return -1;
 
-		return (long) kcopy((char*) r0, (char*) r1, r2);
+//		return (long) kcopy((char*) r0, (char*) r1, r2);
 	case SYSCALL_LS:
 		os_printf("Ls system call called!\n");
-
-		return (long) kls((char*) r0);
+		return -1;
```

Comment:
Maybe add a comment in this file explaining as to why the syscalls don't work anymore.